dashboard/app/views/pd/professional_learning_landing/_pl_extra_links.haml

```diff
@@ -0,0 +1,17 @@
+- extra_links = []
+
+
```

Comment:
I decided to keep the conditions for which links are shown the same as they are on [the homepage](https://github.com/code-dot-org/code-dot-org/blob/daf0ae6d9ed8e3044a6e8ca89ab0edb278497b76/dashboard/app/views/home/_homepage.html.haml#L25), but limit this box to admins and workshop admins. Likely some redundancy here, so I'm open to push back